pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -205,6 +205,7 @@ func (m *manager) Start(activePods ActivePodsFunc, sourcesReady config.SourcesRe
 
 	m.allocatableMemory = m.policy.GetAllocatableMemory(m.state)
 
+	klog.V(4).InfoS("memorymanager started successfully", "policy", m.policy.Name())
```

Comment:
similar comments to https://github.com/kubernetes/kubernetes/pull/129685/files/110868691b271a7448e613f10032b150d23f429e..080962a3a788c54d7f272947d5f4f61903875520#diff-c11864857fa45d5c2d29c3a85d753ccd2db31cca1c7fa046b7144e71f6e59029